### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by eduardostalinho**
Addresses changes requested by @lamenezes in order to fix documentation.

### CODE REVIEW

**Comment by lamenezes** [comment]
I think it would be nice to also fix the import on line 20 (https://github.com/olist/pactum/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R20)

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ is `pactum` package itself and some knowledge of python.
 ```python
 import pactum
 
-class MyAPI(API):
+class MyAPI(pactum.API):
```
Comment (eduardostalinho) [inline README.md]: @lamenezes I implemented the same import just using the name pactum.API just to show that the library can be used with just one import.

File: README.md [hunk 2]
```diff
@@ -102,8 +102,8 @@ if the object is defined by class definition.
 A route can have a list of actions in an HTTP path.
 ```python
 class ResourceRoute(Route):
```
Comment (jpbonson) [inline README.md]: since 'Resource' was changed to 'Orders', shouldn't the class name be changed to something like OrdersRoute?

Comment (eduardostalinho) [reply README.md]: Done. Also did it with API.

**Pull request merged**